src/ui-config/networks.ts

```diff
@@ -194,20 +194,24 @@ export const networkConfigs: Record<string, BaseNetworkConfig> = {
     publicJsonRPCUrl: ['https://mainnet.aurora.dev'],
     publicJsonRPCWSUrl: 'wss://mainnet.aurora.dev',
     addresses: {
-      walletBalanceProvider: '0xDaCbEE6eF2621aeAe30f45A4c2b25F3F15621564',
-      uiPoolDataProvider: '0x83f4137fD99f7Bc47dBb772FC66958A44f9ea34c',
-      uiIncentiveDataProvider: '0x3B45a725d7Db411f0C6bdab5a8ff0B43C73ddCCc',
-      chainlinkFeedRegistry: '0xa305B599892636F59395595e5877079eDa803e72',
+      walletBalanceProvider: '0xa53578865b449db121381bb94d139491f821367c',
+      uiPoolDataProvider: '0x526339dc716acdb92daeac259bd6b0eba4d645d8',
+      uiIncentiveDataProvider: '0xdef68c10f45be945075ef638dd532b846acdfb0d',
+      chainlinkFeedRegistry: '0x0ff4cb8d4ba88bee6d99a3a8493fa77f515c872e',
+      uiIncentiveDataControllers: [
+        '0x181a0c488e3609c7ca95dd4e46dee232db53eb9e'.toLowerCase(),
+        '0xe9eb9bf860d874f7e34ef30d42e6e9c510c19ff7'.toLowerCase(),
+      ],
     },
     protocolDataUrl: 'https://api.thegraph.com/subgraphs/name/pret-labs/protocol-aurora',
     cachingServerUrl: 'https://cache-api-aurora.pret.app/graphql',
     cachingWSServerUrl: 'wss://cache-api-aurora.pret.app/graphql',
     baseUniswapAdapter: '0x0',
     baseAsset: 'ETH',
-    baseAssetWrappedAddress: '0xB8AD40DCfc163694F981D2e12704C0a7af9387DF',
+    baseAssetWrappedAddress: '0x88aabf01c7faab947d1570cd993bed8acebf54b2',
     // incentives hardcoded information
     rewardTokenSymbol: 'WNEAR',
-    rewardTokenAddress: '0x729dfF256Ba68502a17C1443B6F49c6B5300E95d',
+    rewardTokenAddress: '0x1a55e008e86ac4c170499e8e8af5d60565e4e453',
```

Comment:
do we need to configure multiple rewards tokens？

Reply:
resolved.